app/src/components/user/AccountIndexPage/ManageAccount/ManageTeams/TeamViewer/BillingDetails/index.js

```diff
@@ -59,6 +75,15 @@ const BillingDetails = ({ teamId, isTeamAdmin }) => {
       )}
 
       <SubscriptionActionButtons isSubscriptionActive={isSubscriptionActive} />
+      {appSumoSubscriptionInfo && (
+        <>
+          <Divider className="manage-workspace-divider" />
+          <div className="title billing-title">Session Replay Lifetime Pro</div>
+          <p className="text-dark-gray billing-subscription-info">
+            {`This workspace has an active Session Replay Lifetime Pro subscription for ${appSumoSubscriptionInfo?.codes?.length} members.`}
```

Comment:
Can we handle `members` substring for single and multiple members?
keep it `members` for more than 1 else `member`